modules/electrophysiology_browser/jsx/electrophysiologySessionView.js

```diff
@@ -320,153 +325,159 @@ class ElectrophysiologySessionView extends Component {
-            <SummaryPanel
-        id={'filename_summary_' + i}
-        data={this.state.database[i].file.summary}
-        />
-        </div>
-        <EEGMontage />
-        <div className='col-md-6 col-lg-4'>
-            <DownloadPanel
-        id={'file_download_' + i}
-        downloads={this.state.database[i].file.downloads}
-        physioFileID={this.state.database[i].file.id}
-        outputType={this.state.database[i].file.output_type}
-        />
-        </div>
-        </div>
-        </EEGLabSeriesProvider>
-        </div>
-        </FilePanel>
-        </div>
-      );
+                  electrodesURL={electrodesURL}
+                  physioFileID={this.state.database[i].file.id}
+                >
+                  <Panel
+                    id='channel-viewer'
+                    title={
+                      'Signal Viewer' + (file.splitData
+                              ? ' [split '+(file.splitData?.splitIndex+1)+']'
+                              : ''
+                      )
+                    }
+                  >
+                  {file.splitData &&
+                    <>
+                      <span
+                        style={{
+                          color: '#064785',
+                          fontWeight: 'bold',
+                          fontSize: '14px',
+                          paddingRight: '15px',
+                        }}
+                      >
+                      Viewing signal split file:
+                      </span>
+                      <a
+                        className={
+                          'btn btn-xs btn-default split-nav'
+                          + (file.splitData.splitIndex === 0
+                            ? ' disabled'
+                            : '')
+                        }
+                        onClick={() => this.getSplitData(
+                          file.id,
+                          i,
+                          file.splitData.splitIndex-1
+                        )}
+                      >
+                      {'<'}
+                      </a>
+                      {splitPagination}
+                      <a
+                        className={
+                              'btn btn-xs btn-default split-nav'
+                            + (file.splitData.splitIndex
+                            === (file.splitData.splitCount-1)
+                                ? ' disabled'
+                                : '')
+                        }
+                        onClick={
+                          () => this.getSplitData(
+                              file.id,
+                              i,
+                              file.splitData.splitIndex+1
+                          )
+                        }
+                      >
+                      {'>'}
+                      </a>
+                    </>
+                  }
+                  <SeriesRenderer
+                    physioFileID={this.state.database[i].file.id}
```

Comment:
@laemtl is this change supposed to be in this PR?

Reply:
#8238 is merged